pygmt/figure.py

```diff
@@ -394,3 +433,33 @@ def _repr_html_(self):
         subplot,
         text,
     )
+
+
+def set_display(method=None, dpi=None, width=None):
+    """
+    Set the display method.
+
+    Parameters
+    ----------
+    method : str
+        Choose from "notebook" (for inline display in Jupyter notebook)
+        or "external" (for displaying preview using the external viewer).
+
+    dpi : int
+        Set the default DPI (dots-per-inch) used for PNG image previews that
+        are inserted into the notebook.
+    """
+    if method == "notebook":
+        SHOW_CONFIG["notebook"] = True
+        SHOW_CONFIG["external"] = False
+    elif method == "external":
+        SHOW_CONFIG["notebook"] = False
+        SHOW_CONFIG["external"] = True
```

Comment:
Rather than have two configurations, can we not just have a single boolean config object? Are there cases when we need to set both `SHOW_CONFIG["notebook"]` and `SHOW_CONFIG["external"]` to `True`, or both of them to `False`?

How about just having `SHOW_CONFIG["external"]`, defaulting to `False` on Jupyter, but `True` on regular *.py scripts.

Reply:
> Are there cases when we need to set both `SHOW_CONFIG["notebook"]` and `SHOW_CONFIG["external"]` to `True`.

Initially, I allowed setting both to `True`, and realized that it makes the codes more complicated, but I'm not sure if users need it.

> or both of them to `False`?

Yes, we need both of them to `False` when building documentation.

Reply:
Could we maybe have a single setting like `SHOW_CONFIG["display_method"]` (or another name) instead? That parameter could have either 'external', 'notebook' or 'sphinx' settings, and there won't be as many if-elif statements.

Reply:
In commit 6e59fa3d2d625a407a974cd17663f8863767b59c, I changed it to `SHOW_CONFIG["method"]`, and possible values are "external", "notebook" and "none" (str type "none", but `None`).